webui/src/pages/user/user-settings-namespaces.tsx

```diff
@@ -0,0 +1,337 @@
+    },
+    namespaceTabWrapper: {
+        textTransform: 'none'
+    },
+    namespace: {
+
+    },
+    addButton: {
+        [theme.breakpoints.down('md')]: {
+            marginLeft: theme.spacing(2)
+        }
+    },
+
+    memberContainer: {
+        flex: 5,
+        padding: theme.spacing(1)
+    },
+    memberListHeader: {
+        display: 'flex',
+        padding: `0 ${theme.spacing(2)}px ${theme.spacing(2)}px`,
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        [theme.breakpoints.down('sm')]: {
+            flexDirection: 'column',
+            alignItems: 'center'
+        }
+    },
+    foundUserListPopper: {
+        zIndex: 2000
+    },
+    foundUserListContainer: {
+        display: 'flex',
+        flexDirection: 'column',
+        width: 350
+    },
+    foundUserContainer: {
+        display: 'flex',
+        height: 60,
+        alignItems: 'center',
+        '&:hover': {
+            cursor: 'pointer',
+            background: theme.palette.grey[100]
+        }
+    }
+});
+
+interface NamespaceTabProps {
+    chosenNamespace: Namespace,
+    onChange: (event: React.ChangeEvent<{}>, value: Namespace) => void,
+    namespaces: Namespace[]
+}
+
+const NamespacesTabs = (props: NamespaceTabProps) => {
+    const theme = useTheme();
+    const classes = makeStyles(namespacesStyle)();
+    const isATablet = useMediaQuery(theme.breakpoints.down('md'));
+    return <Tabs
+        orientation={isATablet ? 'horizontal' : 'vertical'}
+        value={props.chosenNamespace}
+        onChange={props.onChange}
+        variant={isATablet ? 'scrollable' : 'standard'}
+        scrollButtons={isATablet ? 'auto' : 'off'}
+        className={classes.namespaceContainer}>
+        {
+            props.namespaces.map(namespace => {
+                return <Tab
+                    classes={{
+                        root: classes.namespaceTab,
+                        wrapper: classes.namespaceTabWrapper
+                    }}
+                    key={'nmspc-' + namespace.name}
+                    className={classes.namespace}
+                    value={namespace}
+                    label={namespace.name}
+                />;
+            })
+        }
+    </Tabs>;
+};
+
+class UserSettingsNamespacesComponent extends React.Component<UserSettingsNamespacesComponent.Props, UserSettingsNamespacesComponent.State> {
+
+    constructor(props: UserSettingsNamespacesComponent.Props) {
+        super(props);
+
+        this.state = {
+            namespaces: [],
+            members: [],
+            addDialogIsOpen: false,
+            showUserPopper: false,
+            foundUsers: [],
+            popperTarget: undefined,
+            chosenNamespace: undefined
+        };
+    }
+
+    componentDidMount() {
+        this.initNamespaces();
+    }
+
+    render() {
+        return <React.Fragment>
+            {
+                this.state.namespaces.length > 0 && this.state.chosenNamespace ?
```

Comment:
Please display a `DelayedLoadIndicator` (see `user-settings-tokens`) while the JSON data is loading. The "no namespaces available" message shall be shown only after loading has finished.